packages/supermassive/src/executeWithoutSchema.ts

```diff
@@ -1061,6 +1073,22 @@ function resolveAndCompleteField(
   } catch (rawError) {
     const pathArray = pathToArray(path);
     const error = locatedError(rawError, fieldGroup, pathArray);
+
+    const fieldKey = pathToString(info.path);
+    const hookContextMap = exeContext.hookContextMap;
+    for (const [hookKey, hookData] of hookContextMap.entries()) {
+      if (hookKey.startsWith(fieldKey) && hookKey !== fieldKey) {
+        // On error call complete hooks for all the nested fields, that might otherwise be called too late
+        invokeAfterFieldCompleteHook(
```

Comment:
WIP just do demonstrate the idea. Will add call for `afterFieldResolved` hook later

Reply:
done